src/coreclr/src/interop/agilereferences.cpp

```diff
@@ -0,0 +1,56 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+// Runtime headers
+#include <winwrap.h>
+
+#include "comwrappers.h"
+
+#if (NTDDI_VERSION >= NTDDI_WINBLUE)
```

Comment:
I would just bump WINVER as high as necessary so that we do not need this.

Reply:
I would prefer to avoid the version bump. I will simple forward declare it in all circumstances.